src/devices/vdml_usd.cpp

```diff
@@ -21,5 +21,9 @@ std::int32_t is_installed(void) {
 	return usd_is_installed();
 }
 
+int32_t list_files(const char* path, char* buffer, int32_t len) {
+	return list_files(path, buffer, len);
```

Comment:
umm this is just infinitely recursing right? I think you need `c::list_files(...)`?

Reply:
This is not infinitely recursing. there is `using namespace pros::c;` on line 18 of this file. I have verified it works on hardware